baseplate/sidecars/trace_publisher.py

```diff
@@ -99,9 +99,9 @@ def publish(self, payload: SerializedBatch) -> None:
                 response = getattr(exc, "response", None)
                 if response is not None:
                     logger.exception("HTTP Request failed. Error: %s", response.text)
-                    # If client error, crash
+                    # If client error, skip retries
```

Comment:
:no_bell: I almost asked for a metric, but checked the collapsed diff above and we've already got one. So that's good and means we can make sure people alert on the `error.http` counter to be sure their traces aren't getting quietly dropped. :+1:

Reply:
Do you think a separated counter for client errors would be useful?

Reply:
hm, maybe. "our fault" vs "their fault" errors? can't hurt i suppose

Reply:
Added `errors.http.client` counter.